source/server-discovery-and-monitoring/tests/errors/non-stale-topologyVersion-greater-NotMaster.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Non-stale topologyVersion greater NotMaster error",
+  "description": "Non-stale topologyVersion greater NotWritablePrimary error",
```

Comment:
Many of these files still have "Master" in the name.

Reply:
Good point. I'll check and update the filenames.

Reply:
Pushed filename changes to the PR branch. I've left the filenames as-is in `legacy-hello`.